### DISCUSSION TITLE
Market buyers behavior after reconfiguration

### DISCUSSION THREAD

**Discussion by danieleb2000** [Keriew/augustus#477] [Q&A]
Is the market reconfiguration related only to selling, or also to buying?
For instance: if a market is reconfigured to trade only Furniture, do the buyers search for and buy only Furniture, or like the original they will still try at first to fill Food and Pottery?

**Comment by Keriew** [comment]
Setting a good as not accepted by market means that it won't get any more of it. If it got any before, it will still be distributed.

**Reply by catalin-duta** [reply]
Is this still the expected behaviour now?
I have several markets that I want to rebuild elsewhere. I set everything to "Not trading" and some years later, the market still has the same amount of food and goods, so it seems that it's not only not buying anymore, but also not selling anymore.
This is with build 3.0.1.280.

**Reply by Keriew** [reply]
It's not, it's been changed in af7a3db. I think it's more what you could expect with the option, especially since distributing some goods (e.g. wine) when it's not desired can have a volatile effect.
